rfcs/0095-empowering-new-scoped-module.md

```diff
@@ -0,0 +1,74 @@
+
+# Proposal: Empowering S0 Modules
+
+In this process RFC, We’d like to propose a process to encourage S0 modules and set expectations about what we anticipate in such inclusion.
+
+Note that this RFC focuses on the S0 stage. We propose the following guidelines to expedite to process while ensuring, quality and community support:
+
+- More than three PMC members endorse the S0-level proposal to ensure that there are enough eyes and shepherding in the module. The decision to establish a S0-level module needs to get majority support from PMC.
+- The code changes of S0-level modules follow the normal code review process as in all other modules in the codebase.*
+- A clear set of community members are committed to maintaining the proposed modules with technical support quantitatively, more than three endorsing committers who can serve as the initial owner.
+- No implication that everybody has to immediately work on or switch to the new S0 module.
+- We expect discussions of the relation of the proposed module with existing ones and reuse when possible, but we do not enforce hard no-overlap rules at S0 stage, as most OSS projects do not require modules to have zero perceived overlap.
+- Relations to existing modules and interaction are being clearly discussed, but no hard requirements on zero duplications as per practices in other projects
+- Clean isolation of changes from existing modules, when the change touches existing modules, they should be discussed separately.
+- In discussions of S0-level RFC, maintain a clear separation from S1, and S2 level decisions in later stages so we can encourage S0 changes early while enabling informed decisions at S1, and S2 levels in continued discussions as the modules continue to evolve in the ecosystem.
+- There should be discussions about how the proposal fits into the project to bring clarity. We also acknowledge that not all S1, S2 level decisions can be made at the beginning. Additionally, an S0-module should show a clear positive fit to some(but not all) aspects of the project and clear cohesion to some of the existing modules. As the development evolves, more discussions will happen in future RFCs with additional evidence that help us to make informed decisions.
+
+After the RFC discussion period. One of the PMC members would serve as a champion, provide a clear technical summary of the state, pros and cons during discussions for the S0-level proposal and suggest a path forward. The champion will also continue to drive the overall process of code upstreaming and follow-up discussions.
```

Comment:
Is there any guide about how to get this PMC member, assuming the S0 RFC is not proposed by a PMC member? Also what should we do if the champion doesn't response on time during the upstreaming process? Should the owner find another PMC/committer for help?